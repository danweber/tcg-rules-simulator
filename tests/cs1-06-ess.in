VERSION:2
# [Your Turn] When this Monster attacks your opponent's Monster, suspend one of your opponent's Blue Monsters.
FIELD1:Sneaky,DW1-Van,CS1-06
FIELD2:White,DW1-Van Blue,CS2-02 Victim,CS2-03,REST
SECURITY2:CS2-03
GAME:P1 T2 MAIN 7
DUMP
ECHO: -- Attack 

CMD:ATTACK Sneaky Victim
STEP:AUTOANSWER:CAPTURE
DUMP
EXIT
