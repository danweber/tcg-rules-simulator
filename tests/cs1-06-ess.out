FIELD1:Sneaky,DW1-Vanillamon,CS1-Sneakycat
FIELD2:White,DW1-Vanillamon Blue,CS2-Sharpblade Victim,CS2-Longsword,REST
GAME:P1 T2 MAIN 7
ECHO: -- Attack
CAPTURE:Target Sharp Blade 3
CAPTURE:Target Long Sword 4
FIELD1:Sneaky,DW1-Vanillamon,CS1-Sneakycat,REST
FIELD2:White,DW1-Vanillamon Blue,CS2-Sharpblade,REST
GAME:P1 T2 MAIN 7
